js/controller.js

```diff
@@ -58,7 +58,9 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         "countDownFinished": false,
         "countDownCancelled": false,
         "timeToShow": 0,
-        "showing": false
+        "showing": false,
+        "delayedSetEmbedCodeEvent": false,
+        "selectedContentData": null
```

Comment:
- Weird naming, "selectedContentData" can be confusing to developer.
- selectedContentData is only being used if delayedSetEmbedCodeEvent is set to TRUE. Do you agree to refactor these 2 variables?